docs/adr/0014-use-mux-for-routing.md

```diff
@@ -0,0 +1,53 @@
+# Use gorilla/mux for Routing
+
+We need a way to serve routes so that we can create an application with functionality
+that is useful for the front end.
+
+## Considered Alternatives
+
+* [gorilla/mux](https://github.com/gorilla/mux)
+* [gin-gonic/gin](https://github.com/gin-gonic/gin)
+* Use the Standard Library Only
+* Write our own routing
+
+## Decision Outcome
+
+* Chosen Alternative: [gorilla/mux](https://github.com/gorilla/mux)
+* We've decided that gorilla/mux is the right solution
+as it is maintained by [Gorilla Tool Kit](https://www.gorillatoolkit.org/).
```

Comment:
Is the main reason for picking the package who develops it? I see this more as a single bullet of "developed by a reputable team in the community"